float32[6] efforts  # amps
float32[6] vels     # rad/s
float32[6] pos      # rad
uint32 msgCounter   # counter to check for missed messages